files/archive.py

```diff
@@ -158,6 +186,22 @@ def main():
         else:
             expanded_paths.append(path)
 
+    # Only attempt to expand the exclude paths if it exists
+    if exclude_paths:
+        for i, exclude_path in enumerate(exclude_paths):
+            exclude_path = os.path.expanduser(os.path.expandvars(exclude_path))
+
+            # Expand any glob characters. If found, add the expanded glob to the
+            # list of expanded_paths, which might be empty.
+            if ('*' in exclude_path or '?' in exclude_path):
+                expanded_exclude_paths = expanded_exclude_paths + glob.glob(exclude_path)
+                globby = True
```

Comment:
I don't think we want globby to be set here, since it will force the output as an archive, even if you specified a single path to compress.

I.e., 

`archive: path=/a/single/file.txt exclude_paths=/globby/path/*`

Will cause this to fail as 'archive will get set on L166, but 'dest' is not defined.

I think just removing this line is fine – `globby` indicates that we _want_ an archived file, but the usage above does not indicate that.